src/app/auto-suggest/auto-suggest.component.html

```diff
@@ -10,12 +10,9 @@
   <ng-container *ngFor="let suggestion of suggestions">
     <div class="suggestion" (click)="addLocation(suggestion.item)">
       {{ suggestion.item.location }}
-      <a
-        [href]="suggestion.item.source.url"
-        target="_blank"
-        rel="noopener noreferrer"
-        >(Source: {{ suggestion.item.source.name }})</a
-      >
+      <span class="source">
+        (Source: {{ suggestion.item.source.name }})
+      </span>
```

Comment:
I like linking to sources -- let's do it elsewhere though (maybe an infobox on the bottom of the page). Doesn't need to be inside the dropdown (I keep accidentally clicking it instead of the location)

Reply:
agreed